### PR TITLE
Added typography

### PR DESCRIPTION
**Pull request by rutu14**
Preview: https://deploy-preview-12--friendly-kilby-44e498.netlify.app/documentation/typography-heading.html

### CODE REVIEW

File: documentation/typography-heading.html [hunk 1]
```diff
@@ -114,14 +117,44 @@ <h3 class="heading4 medium">Components</h3>
...
+                            <h1 contenteditable="true" class="heading3 medium edit-text">Everything you can imagine is real.</h1>
+                            <h1 contenteditable="true" class="heading4 medium edit-text">Everything you can imagine is real.</h1>
+                            <h1 contenteditable="true" class="heading5 medium edit-text">Everything you can imagine is real.</h1>
+                        </div>
+                    </div> 
+                    <div id="panel-12" role="tabpanel" aria-labelledby="tab-12" hidden>
+                        <div class="code-syntax">
+                            <script type="text/plain" class="line-numbers language-markup">
+<h1 class="heading1 medium">...</h1>       
+<h1 class="heading2 medium">...</h1>
```
Comment (Ranjeetk489) [inline documentation/typography-heading.html:147]: spacing is uneven

File: documentation/typography-text.html [hunk 2]
```diff
@@ -114,13 +117,43 @@ <h3 class="heading4 medium">Components</h3>
...
+                            <p contenteditable="true" class="text3 regular edit-text">And so the adventure begins.</p>
+                            <p contenteditable="true" class="text4 regular edit-text">And so the adventure begins.</p>
+                            <p contenteditable="true" class="text5 regular edit-text">And so the adventure begins.</p>
+                        </div>
+                    </div> 
+                    <div id="panel-12" role="tabpanel" aria-labelledby="tab-12" hidden>
+                        <div class="code-syntax">
+                            <script type="text/plain" class="line-numbers language-markup">
+<p class="text1 regular">...</p>       
+<p class="text2 regular">...</p>
```
Comment (Ranjeetk489) [inline documentation/typography-text.html:146]: spacing is uneven

**Review by Ranjeetk489 — APPROVED** [review]
make minor edits and after that it's Good to merge

File: documentation/css/style.css [hunk 3]
```diff
@@ -130,7 +130,7 @@
     grid-area: aside;
     display: flex;
     flex-direction: column;
-    padding: 1.6rem;
+    padding:1.6rem 1.6rem 2rem;
     box-shadow: 0 0.2rem 0.5rem 0 var(--box-shadow), 0 2px 10px 0 var(--box-shadow);
```
Comment (zoef-yt) [inline documentation/css/style.css:134]: use px here instead of rem

**Review by zoef-yt — APPROVED** [review]
Good to merge

**Review by uday-nv-work — APPROVED** [review]
Good to merge

File: documentation/css/style.css (hunk 3 above)
Comment (rutu14) [reply documentation/css/style.css:134]: Cool

File: documentation/typography-heading.html (hunk 1 above)
Comment (rutu14) [reply documentation/typography-heading.html:147]: It is for syntax highlighter.

**Pull request merged**